imu_filter_madgwick/src/imu_filter_ros.cpp

```diff
@@ -176,10 +211,12 @@ void ImuFilterRos::imuCallback(const ImuMsg::ConstPtr& imu_msg_raw)
     dt = constant_dt_;
   else
   {
-    dt = (time - last_time_).toSec();
-    if (time.isZero())
-      ROS_WARN_STREAM_THROTTLE(5.0, "The IMU message time stamp is zero, and the parameter constant_dt is not set!" <<
+    dt = (time - last_time_).seconds();
+    if (0 == time.nanoseconds()) {
+      rclcpp::Clock steady_clock(RCL_STEADY_TIME);
```

Comment:
Wouldn't it make more sense to have a single `static rclcpp::Clock steady_clock(RCL_STEADY_TIME)` at the beginning of this method instead of re-declaring a new clock multiple times?

Reply:
Done